bycycle/plts/cyclepoints.py

```diff
@@ -0,0 +1,99 @@
+"""Plot extrema and zero-crossings."""
+
+import numpy as np
+from scipy.stats import zscore
+
+import matplotlib.pyplot as plt
+from matplotlib import rcParams
+
+from neurodsp.plts import plot_time_series
+
+from bycycle.plts.utils import apply_tlims, get_extrema
+
+###################################################################################################
+###################################################################################################
+
+def plot_cyclepoints(df, sig, fs, tlims=None, ax=None, plot_sig=True,
+                     plot_extrema=True, plot_zerox=True, **kwargs):
+    """Plot extrema and/or zero-crossings.
+
+    Parameters
+    ----------
+    df : pandas.DataFrame
+        Dataframe output of :func:`~.compute_features`.
+    sig : 1d array
+        Time series to plot.
+    fs : float
+        Sampling rate, in Hz.
+    tlims : tuple of (float, float), optional, default: None
+        Start and stop times.
+    ax : matplotlib.Axes, optional, default: None
+        Figure axes upon which to plot.
+    plot_sig : bool, optional, default: True
+        Plots the raw signal.
+    plot_extrema :  bool, optional, default: True
+        Plots peaks and troughs.
+    plot_zerox :  bool, optional, default: True
+        Plots zero-crossings.
+    **kwargs
+        Keyword arguments to pass into `plot_time_series`.
+
+    Notes
+    -----
+    Default keyword arguments include:
+
+    - ``figsize``: tuple of (float, float), default: (15, 3)
+    - ``xlabel``: str, default: 'Time (s)'
+    - ``ylabel``: str, default: 'Voltage (uV)
+
+    """
+
+    rcParams['lines.markersize'] = 12
```

Comment:
Won't this have side-effects? Like, effecting all subsequent plots? I'm not sure we want to be updating rcParams in here (though I am also a little naive to it's scope / how it works - so maybe it doesn't matter).